packages/react/src/BranchName/BranchName.tsx

```diff
@@ -19,5 +23,34 @@ const BranchName = styled.a<SxProp>`
   ${sx};
 `
 
-export type BranchNameProps = ComponentProps<typeof BranchName>
+type BranchNameProps<As extends React.ElementType> = {
+  as?: As
+} & React.ComponentPropsWithoutRef<React.ElementType extends As ? 'a' : As> &
+  SxProp
+
+function BranchName<As extends React.ElementType>(props: BranchNameProps<As>) {
+  const {as: BaseComponent = 'a', className, children, sx, ...rest} = props
+  const enabled = useFeatureFlag('primer_react_css_modules_team')
+  if (enabled) {
+    if (sx) {
+      return (
+        <Box {...rest} as={BaseComponent} className={clsx(className, classes.BranchName)} sx={sx}>
```

Comment:
I think it's somehow always ending up in this path? Noticing there is styled components classes still even when there's no sx styles

Reply:
ah ok, was looking at some old storybook